Customized App so that we can add Header/Footer based on a config

razvanMiu opened the pull request.

razvanMiu committed:
Customized App so that we can add Header/Footer based on a config

razvanMiu committed:
Get theme from route data

razvanMiu committed:
Removed volto-addons dependency

tiberiuichim reviewed (commented):
Use Volto's InlineForm here, I think we're using BlockForm just because it predates InlineForm.

silviubogan committed:
Solve 2 yarn start errors

tiberiuichim committed:
Merge pull request #55 from eea/solve-yarn-start-errors

Solve 2 yarn start errors

silviubogan committed:
Remove customization of Html.jsx for critical CSS

The customization is not needed anymore as the optimization for reducers
is included in Volto and Volto also uses critical CSS in its Html.jsx
file.

tiberiuichim committed:
Merge pull request #56 from eea/use-critical-css

Remove customization of Html.jsx for critical CSS in Volto 11

silviubogan committed:
Remove ImageCards block in favour of separate addon

at: https://github.com/eea/volto-block-image-cards

tiberiuichim committed:
Merge pull request #57 from eea/remove-old-imagecards

Remove ImageCards block in favour of separate addon
